app/src/main/res/values-de/strings.xml

```diff
@@ -239,12 +239,153 @@
     <string name="recover_safe_info_explainer">Bitte halte deine Safe-Adresse sowie die 12 Wörter deines Sicherungsschlüssels bereit.</string>
     <string name="start_recovery">Starte Wiederherstellung</string>
     <string name="connected_extension">Verbundene Extension</string>
-    <string name="connect_browser_extension">Verbinde Browser Extension</string>
+    <string name="connect_browser_extension">Verbinde Browser\nExtension</string>
     <string name="connect_extension_description">Diese Transaktion verbindet diesen Safe mit der Browser Extension. Zukünftige Transaktionen müssen mit beiden Geräten bestätigt werden.</string>
     <string name="pairing_first_step"><![CDATA[1. Lade die <font color=#27a0ff>Gnosis Safe Browser Extension</font> herunter.]]></string>
     <string name="pairing_second_step">2. Scanne den QR-Code in der Browser Extension.</string>
     <string name="submitted">Abgesendet</string>
     <string name="status">Status</string>
     <string name="transaction_fee">Transaktionsgebühr</string>
     <string name="later">Später</string>
+    <string name="payment_token"></string>
```

Comment:
We should not add keys where we don'T have transactions

Reply:
Removed empty strings